src/index.ts

```diff
@@ -93,7 +92,7 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     .getConfiguration("metals")
     .get<string[]>("serverProperties")!;
 
-  const javaOptions = getJavaOptions();
+  const javaOptions = getJavaOptions(workspace.workspaceFolders[0]!.uri);
```

Comment:
Since we're doing `workspace.workspaceFolders[0]!.uri` could we just do `workspace.workspaceFolder!.uri` to return the current workspace uri.

Reply:
Sure, good idea!